lstests/core/service.py

```diff
@@ -0,0 +1,1522 @@
+        - send_raw()
+        - pause_node_production()
+        - resume_node_production()
+        - get_greylist()
+        - add_greylist_accounts()
+        - remove_greylist_accounts()
+        - get_net_plugin_connections()
+    """
+    def __init__(self,
+                 service,
+                 cdir=None,
+                 cluster_id=None,
+                 node_count=None,
+                 pnode_count=None,
+                 producer_count=None,
+                 unstarted_count=None,
+                 topology=None,
+                 center_node_id=None,
+                 tokens_supply=None,
+                 extra_configs: typing.List[str]=None,
+                 extra_args: str=None,
+                 dont_newacco=None,
+                 dont_setprod=None,
+                 http_retry=None,
+                 http_sleep=None,
+                 verify_async=None,
+                 verify_retry=None,
+                 verify_sleep=None,
+                 sync_retry=None,
+                 sync_sleep=None):
+        """Create a Cluster object to launch a cluster of nodes.
+
+        The node cluster is the plaform on which a test will be performed.
+
+        Parameters
+        ----------
+        service : Service
+            Launcher service object on which the node cluster will run
+        cidr : str
+            Smart contracts directory.
+            Can be either absolute or relative to service's working directory.
+        cluster_id : int
+            Cluster ID to launch with.
+            Tests with different cluster IDs can be run in parallel.
+            Valid range is [0, 30). Default is 0.
+        node_count : int
+            Number of nodes in the cluster.
+            Default is 4.
+        pnode_count : int
+            Number of nodes with at least one producer.
+            Default is 4.
+        producer_count : int
+            Number of producers in the cluster.
+            Default is 4.
+        unstarted_count : int
+            Number of unstarted nodes.
+            Default is 0.
+        topology : str
+            Cluster topology to launch with.
+            Valid choices are "mesh", "bridge", "line", "ring", "star", "tree".
+            Default is "mesh".
+        center_node_id : int
+            Center node ID (for bridge or star topology).
+            If topology is bridge, center node ID cannot be 0 or last one.
+            No default value.
+        extra_configs : list
+            Extra configs to pass to nodeos via launcher service.
+            e.g. ["plugin=SOME_EXTRA_PLUGIN"]
+            No default value.
+        extra_args : str
+            Extra arguments to pass to launcher service.
+            e.g. "--delete-all-blocks"
+            No default value.
+        dont_newacco : bool
+            Do not create accounts in launch.
+            Default is False (will create producer accounts).
+        dont_setprod : bool
+            Do not set producers in BIOS launch mode.
```

Comment:
"set producers in BIOS launch mode." -> "set producers."  BIOS launch mode is the only available mode now.

Reply:
Thanks for the catch. Updated. Thank you.